addons/mail_connector/views/mail_connector_config_view.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="mail_connector_config_settings_view" model="ir.ui.view">
+        <field name="name">Mail Connector Config Settings</field>
+        <field name="model">mail.connector.config.settings</field>
+        <field name="arch" type="xml">
+            <form string="Configure Channel" class="oe_form_configuration">
+                <header>
+                    <button string="Apply" type="object" name="execute" class="oe_highlight" />
+                    <button string="Cancel" type="object" name="cancel" class="oe_link" />
```

Comment:
type="object" in cancel button ? we have keyword use special="cancel" no need to define type.